docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+* *destination2*: `topology.liqo.io/type=destination`
+
+You can check that the clusters are correctly labeled through:
+
+```bash
+liqoctl status
+liqoctl status --kubeconfig "$KUBECONFIG_DESTINATION1"
+liqoctl status --kubeconfig "$KUBECONFIG_DESTINATION1"
+```
+
+## Peer the clusters
+
+Once Liqo is installed in your clusters, you can establish new *peerings*.
+In this example, you will use the [in-band peering approach](FeaturesPeeringInBandControlPlane).
+
+To establish the peerings from *origin* to *destination* clusters run:
+
+```bash
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION1"
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION2"
+```
+
+When the above commands return successfully, you can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that an outgoing peering is currently active towards both the *destination1* and the *destination2* clusters, as well as the cross-cluster network tunnels have been established:
+
+```text
+NAME           TYPE     OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+destination1   InBand   Established        None               Established   Established      41s
+destination2   InBand   Established        None               Established   Established      7s
+```
+
+Additionally, you should have two new virtual nodes in the *origin* cluster, characterized by the install-time provided labels:
+
+```bash
+kubectl get node --selector=liqo.io/type=virtual-node --show-labels
+```
+
+```text
+NAME            STATUS   ROLES   AGE     VERSION   LABELS
+NAME                STATUS   ROLES   AGE   VERSION   LABELS
+liqo-destination1   Ready    agent   57s   v1.25.0   liqo.io/remote-cluster-id=df0baf71-c428-4d2e-8c97-3559e205d34a,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+liqo-destination2   Ready    agent   43s   v1.25.0   liqo.io/remote-cluster-id=3c4604aa-b9b4-4442-be6d-8ffc713139fc,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+```
+
+```{admonition} Note
+Some of the default labels were omitted for the sake of clarity.
+```
+
+## Tune namespace offloading
+
+Now, let's suppose you want to deploy an application that needs to be scheduled on every *destination* cluster, but not in the *origin* one.
+
+```bash
+kubectl create namespace liqo-demo
+```
+
+Then, enable Liqo offloading for that namespace:
+
+```bash
+liqoctl offload namespace liqo-demo \
+  --namespace-mapping-strategy EnforceSameName \
+  --pod-offloading-strategy Remote \
+  --selector 'topology.liqo.io/type=destination'
+```
+
+The above command configures the following aspects (see the dedicated [usage page](/usage/namespace-offloading.md) for additional information concerning namespace offloading configurations):
+
+* the `liqo-demo` namespace, and the contained resources, are offloaded only to the clusters with the `topology.liqo.io/type=destination` label.
```

Comment:
I would replace it with a box saying something like "Pro Tip: you can select a subset of your cluster as explained in the [Offloading with Policies](https://docs.liqo.io/en/v0.6.1/examples/offloading-with-policies.html) demo"